analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -964,7 +970,6 @@ public void Invocation_Equals_SplitsToBothResults_Nullable(string left, string r
             var validator = SETestContext.CreateCS(code).Validator;
             var result = validator.Symbol("result");
             validator.TagStates("End").Should().SatisfyRespectively(
-                x => x[result].HasConstraint(BoolConstraint.True).Should().BeTrue(),
```

Comment:
Se the `ToDo` on the `DataRow`. This whole test method should be removed. The `DataRow` should be moved to another test method

Reply:
Ah. I understood you wrong. I moved the test case in 48f730e23fe6406c8e19287b22821d1a0d448e1f. Please have a second look.